src/sqerl_pgsql_client.erl

```diff
@@ -178,23 +190,23 @@ init(Config) ->
             false -> undefined
         end,
     case pgsql:connect(Host, User, Pass, Opts) of
-        {error, timeout} ->
-            {stop, timeout};
```

Comment:
The code in question (https://github.com/epgsql/epgsql/blob/master/src/pgsql.erl#L73-L82) looks like it will either return `{ok, Connection}` or `{error, Something}` and we handle `{error, Something}` below